src/object/distance.rs

```diff
@@ -75,7 +75,10 @@ impl Topology {
     /// one of them is returned. The same applies for `MEANS_xyz` options.
     #[allow(clippy::missing_errors_doc)]
     #[doc(alias = "hwloc_distances_get")]
-    pub fn distances(&self, kind: DistancesKind) -> Result<Vec<Distances<'_>>, RawHwlocError> {
+    pub fn distances(
+        &self,
+        kind: Option<DistancesKind>,
```

Comment:
Since distance kind is a filter, it makes sense to have an easy None value at hand.